micrometer-shim/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry Micrometer Bridge"
+otelJava.moduleName.set("io.opentelemetry.micrometershim")
+
+dependencies {
+  api("io.micrometer:micrometer-core")
```

Comment:
In instrumentation repo this is `compileOnly` but here went with `api` for now to match pattern in opentracing shim but may change it.